cmd/wallet-sdk-gomobile/walleterror/error.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright Avast Software. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package walleterror define error model for go mobile api.
+package walleterror
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/trustbloc/wallet-sdk/pkg/walleterror"
+)
+
+// Error represents error returned by go mobile api.
+type Error struct {
+	Code    string `json:"code"`
+	Error   string `json:"error"`
```

Comment:
It's not clear from looking at this why the `Error` struct also has a field named `Error`. What is `Error.Error` supposed to be? Is it more like an error type/category? If so, I would suggest `Type` or `Category` (`Type` might be a reserved word in the bindings?)

Reply:
done, now named Category